owncloudApp/src/main/java/com/owncloud/android/ui/preview/PreviewImageFragment.kt

```diff
@@ -347,6 +358,7 @@ class PreviewImageFragment : FileFragment() {
         private const val ARG_FILE = "FILE"
         private const val ARG_ACCOUNT = "ACCOUNT"
         private const val ARG_IGNORE_FIRST = "IGNORE_FIRST"
+        private const val SVG_MIMETYPE = "image/svg+xml"
```

Comment:
I think that this mimetype could be moved to https://github.com/owncloud/android/blob/master/owncloudDomain/src/main/java/com/owncloud/android/domain/files/MimeTypeConstants.kt to keep the mimetypes together. What do u think?

Reply:
I was unaware of the existence of this class, perfect. Changed.